client/controllers/events/eventIncidentReports.coffee

```diff
@@ -139,6 +139,10 @@ Template.eventIncidentReports.onRendered ->
       @updatePlot(groups)
       return
 
+  @autorun =>
+    EventIncidents.find(Template.instance().data.filterQuery.get())
+    $('tr.details').remove()
```

Comment:
I think it should be separate. Although the dependencies are currently the same that might not always be the case because the autoruns control separate components.